website/source/docs/jobops/logs.html.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: "docs"
+page_title: "Operating a Job: Accessing Logs"
+sidebar_current: "docs-jobops-logs"
+description: |-
+  Learn how to operate a Nomad Job.
+---
+
+# Accessing Logs
+
+Accessing applications logs is critical when debugging issues, performance
+problems or even for checking the application is starting correctly. To make
```

Comment:
s/checking/verifying/.  Checks should be done by systems, verification can be done by administrators.